pom.xml

```diff
@@ -80,14 +80,14 @@
   </repositories>
 
   <properties>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
     <junit.version>4.11</junit.version>
     <commons-lang.version>2.6</commons-lang.version>
     <guava.version>19.0</guava.version>
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
     <app.parents>
-      system:cdap-etl-batch[4.0.0,5.0.0-SNAPSHOT),system:cdap-etl-realtime[4.0.0,5.0.0-SNAPSHOT),system:cdap-data-pipeline[4.0.0,5.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,5.0.0-SNAPSHOT)
+      system:cdap-etl-batch[6.0.0,7.0.0-SNAPSHOT),system:cdap-etl-realtime[6.0.0,7.0.0-SNAPSHOT),system:cdap-data-pipeline[6.0.0,7.0.0-SNAPSHOT),system:cdap-data-streams[6.0.0,7.0.0-SNAPSHOT)
```

Comment:
nit: So far I have seen it was `6.0.0-SNAPSHOT - 7.0.0-SNAPSHOT` as version range. Might be something we want to check so that it is consistent.